File: _includes/code/llms-txt/python/local_connection.py
"""llms.txt snippet: local connection. Section "Python / TypeScript > Local connection"."""
import weaviate

# START llms_local_connection
client = weaviate.connect_to_local()  # localhost:8080, gRPC 50051
# END llms_local_connection

assert client.is_ready()
client.close()

File: _includes/code/llms-txt/typescript/local_connection.ts
// llms.txt snippet: local connection. Section "Python / TypeScript > Local connection".
import weaviate from 'weaviate-client';

// START llms_local_connection
import weaviate from 'weaviate-client';

const client = await weaviate.connectToLocal();
// END llms_local_connection

if (!(await client.isReady())) throw new Error('Weaviate not ready');
await client.close();
